packages/teams-js/src/public/dialog.ts

```diff
@@ -125,7 +125,7 @@ export namespace dialog {
    * @beta
    */
   export function submit(result?: string | object, appIds?: string | string[]): void {
-    ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.task, FrameContexts.meetingStage);
+    ensureInitialized(FrameContexts.content, FrameContexts.task);
```

Comment:
>FrameContexts.content [](http://example.com/codeflow?start=22&length=21)

Is there a bug and/or timeline for when Teams mobile will be fixing this and when we can remove this?

Reply:
I have asked for the timeline. If I don't get a clear answer, I will pass the bug tracking part to our PMs who are tracking other bugs in iOS as well.